mobile/lib/features/wallet/seed_phrase_import_confirmation.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/material.dart';
+import 'package:get_10101/common/global_keys.dart';
+import 'package:get_10101/common/loading_screen.dart';
+import 'package:get_10101/common/snack_bar.dart';
+import 'package:get_10101/features/wallet/seed_phrase_importer.dart';
+import 'package:get_10101/features/wallet/seed_words.dart';
+import 'package:get_10101/ffi.dart';
+import 'package:get_10101/logger/logger.dart';
+import 'package:get_10101/util/file.dart';
+import 'package:go_router/go_router.dart';
+
+class SeedPhraseImportConfirmation extends StatelessWidget {
+  static const route = "${SeedPhraseImporter.route}/$subRouteName";
+  static const subRouteName = "seed-import-confirmation";
+
+  final List<String> seedWords;
+
+  const SeedPhraseImportConfirmation({super.key, required this.seedWords});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(title: const Text('Import Seed')),
+        body: Column(
+          children: [
+            buildSeedWordsWidget(seedWords, true),
+            const SizedBox(height: 20),
+            Row(
+              mainAxisAlignment: MainAxisAlignment.end,
+              children: <Widget>[
+                ElevatedButton(
```

Comment:
I guess we don't need the cancel button given this is wrapped in an Scaffold with an AppBar which has a back button.

Reply:
You technically don't need it, but it's more obvious to the user this way that they can cancel. 
Not everyone will immediately notice that there's an AppBar on the first run of the app - not all platforms use that way of navigation.